src/components/pages/email-change/EmailChange.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+
+import LoaderContainer from '../../../components/layout/Loader/LoaderContainer'
+import { API_URL } from '../../../utils/config'
+import { parse } from 'query-string'
+import { toast } from 'react-toastify'
+
+export const REQUEST_EMAIL_CHANGE_PAGE_LINK = '/profil/email'
+export const SIGNIN_PAGE_LINK = '/connexion'
+export const EMAIl_CHANGE_FAILED = "La modification de l'adresse email a échouée."
+export const EMAIl_CHANGE_LINK_EXPIRED = 'Le lien a expiré, veuillez réessayer.'
+export const EMAIL_CHANGE_SUCCESS = "L'adresse email a bien été modifiée."
+
+const EmailChange = ({ location, history }) => {
+  useEffect(() => {
+    (async () => {
+      const { token, expiration_timestamp } = parse(location.search)
+      const expiration_date = new Date(expiration_timestamp * 1000)
+      if (expiration_date < new Date(Date.now())) {
```

Comment:
```suggestion
      if (expiration_date < new Date()) {
```

Ça l'air équivalent. (Sauf si je loupe un truc un peu sioux en JavaScript. Ce qui n'est pas complètement improbable ! ;)

Reply:
Alternative, comparer des timestamps :

```javascript
if (expiration_timestamp * 1000 < Date.now())
```
(j'ai mis `*1000` comme toi, mais je ne sais pas ce que je fais...)

Reply:
C'est pour pouvoir mocker Date.now, j'avais pris la première syntaxe en effet ;)
Quand à ce qui est de comparer les timestamps, je vais faire ça, pas besoin de passer par des objets Date js en effet !

Reply:
Je peux aussi mettre un petit commentaire pour le *1000 qui convertit en ms le timestamp renvoyé par unix qui est en secondes :)